src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
```

Comment:
Ensure that the `table` element exists in the DOM before querying it. If the table is not present, this could lead to errors.